src/app/dashboard/directory-dashboard/directory-dashboard.component.html

```diff
@@ -1,24 +1,27 @@
 <div class="d-flex flex-column justify-content-center dashboard-content">
     <div class="col-12 pt-4 pb-4 scroll-item">
         <div class="card card-item">
+            <div class="card-header p-1" (click)="collapseDashboard()">
+                <span class="pull-right fa fa-bars"></span>
+            </div>
             <div class="card-body p-2">
-                <app-directory-dashboard-menu></app-directory-dashboard-menu>
-                <hr class="my-1">
-                <app-directory-contact-info></app-directory-contact-info>
-                <hr class="my-1">
-                <app-directory-summary></app-directory-summary>
+                <app-directory-dashboard-menu [dashboardCollapsed]="dashboardCollapsed"></app-directory-dashboard-menu>
```

Comment:
Don't need to assign the collapse class on each individual element. If all elements are getting collapsed, assign collpase to the parent/container